phonenumber_field/phonenumber.py

```diff
@@ -43,9 +43,7 @@ def __str__(self):
 
     def __repr__(self):
         if not self.is_valid():
-            return str(
-                "Invalid{}(raw_input={})".format(type(self).__name__, self.raw_input)
-            )
+            return str(f"Invalid{type(self).__name__}(raw_input={self.raw_input})")
```

Comment:
Do we really need to convert to a string?

Reply:
Addressed in #485.